lib/import-export-visitor.js

```diff
@@ -303,15 +318,16 @@ IEVp.finalizeHoisting = function () {
     delete bodyInfo.hoistedExportsMap;
     delete bodyInfo.hoistedExportsString;
     delete bodyInfo.hoistedImportsString;
-  }, this);
+  }
 
   // Just in case we call finalizeHoisting again, don't hoist anything.
   this.bodyInfos.length = 0;
 
-  this.removals.forEach(processRemoval);
+  let removalCount = this.removals.length;
 
-  // Just in case we call finalizeHoisting again, don't remove anything.
-  this.removals.length = 0;
+  while (removalCount--) {
+    processRemoval(this.removals.pop());
```

Comment:
Going in reverse order could be a meaningful semantic change. Also, calling `.pop()` with each iteration seems like a step in the wrong direction, performance-wise. Can we just keep the `forEach` please? The `processRemoval` function is the same every time.